### PR TITLE
Room show events

### PR DESCRIPTION
**Pull request by rcyep87**
Added grid functionality and Room's show page displays event information. Haven't converted user_id to User's name who makes the reservation.

### CODE REVIEW

**Comment by rcyep87** [comment]
I also forgot to mention I enabled bootstrap in this code.

File: app/views/rooms/show.html.erb [hunk 1]
```diff
@@ -12,8 +12,32 @@
...
+      <th>Duration</th>
+      <th>Description</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @room.events.each do |event| %>
+      <tr>
+        <td><%= event.user_id %></td>
```
Comment (jah2488) [inline app/views/rooms/show.html.erb:32]: We should replace this with the actual users name.

File: app/views/rooms/show.html.erb [hunk 2]
```diff
@@ -12,8 +12,32 @@
...
+        <td><%= event.start_time %></td>
```
Comment (jah2488) [inline app/views/rooms/show.html.erb:33]: We should convert this to a readable timestamp.

**Comment by jah2488** [comment]
:+1:

**Comment by cmccammon** [comment]
:+1:

**Pull request merged**